bootstrap.py

```diff
@@ -0,0 +1,84 @@
+import random
+import threading
+import uuid
+
+from keystoneclient import exceptions
+from keystoneclient.v3 import client
+
+keystone = client.Client(
+    username='admin',
+    password='password',
+    project_name='admin',
+    project_domain_id='default',
```

Comment:
These should use project_domain_name='Default' since we'll likely change keystone to use a uuid for the default domain ID.